### ISSUE TITLE
no-array-prototype-extensions false positive Promsie.withResolvers

### ISSUE THREAD

**Issue by sukima** [ember-cli/eslint-plugin-ember#2360]
## Reproduction

```typescript
const resolvers = Promise.withResolvers();
resolvers.reject();
```

I don't know how to check for that though.
